polaris.shopify.com/src/components/Layout/Layout.module.scss

```diff
@@ -1,119 +1,89 @@
 @import "../../styles/variables.scss";
 
+@mixin layout-scrollbars {
+  &::-webkit-scrollbar {
+    width: 1px;
+  }
+
+  &::-webkit-scrollbar-track {
+    background-color: transparent;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: transparent;
+  }
+
+  &:hover {
+    &::-webkit-scrollbar-thumb {
+      background-color: var(--text-subdued);
+    }
+  }
+}
```

Comment:
Should this go in `mixins.scss`?

Reply:
Maybe, be I don't think any other component than `Layout` should use these scrollbars.